src/pages/validator/amplifier/integrator-onboarding.mdx

```diff
@@ -0,0 +1,87 @@
+# Integrate your chain
+
+## Prerequisites
+
+- Make sure you have `wasm` enabled.
+- You should know how to deploy and interact with a CosmWasm contract. Refer to the [Osmosis CosmWasm testnet deployment tutorial](https://docs.osmosis.zone/cosmwasm/testnet/cosmwasm-deployment/) if you’d like a refresher — just replace `osmosisid` with `axelard` and use the `arm64` version of the [contract optimizer](https://github.com/CosmWasm/optimizer) if you are working on a [Mac with Apple silicon](https://support.apple.com/en-us/116943). Run `--gas auto --gas-adjustment 2` to provide your transactions with sufficient gas.
+
+## Devnet information
+
+- Devnet RPC: `http://devnet.rpc.axelar.dev:26657`
+- Devnet LCD (REST API/JSON RPC): `http://devnet.rpc.axelar.dev:1317`
+- Devnet gRPC: `http://devnet.rpc.axelar.dev:9090`
+- Chain ID: `devnet-wasm`
+- Other info can be found in [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/e6f549de873e0ad211a073218f172d03f6e92c3b/axelar-chains-config/info/devnet-amplifier.json).
```

Comment:
link to the merged config/main branch

Reply:
To this? https://github.com/axelarnetwork/axelar-contract-deployments/commit/e6f549de873e0ad211a073218f172d03f6e92c3b

Is there a reason we want to link to a PR instead of the file itself?

Reply:
needs to be linked to the file on main branch, not a specific commit (so users see the up-to-date values)
https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json